mlflow/tracking/_model_registry/registry.py

```diff
@@ -29,7 +29,7 @@ def get_store(self, store_uri=None):
         :return: An instance of `mlflow.store.model_registry.AbstractStore` that fulfills the
                  store URI requirements.
         """
-        from mlflow.tracking._tracking_service import utils
-        store_uri = store_uri if store_uri is not None else utils.get_tracking_uri()
+        from mlflow.tracking._model_registry import utils
+        store_uri = utils._resolve_registry_uri(store_uri)
```

Comment:
this does change the behavior such that `store_uri = ""` is not a legit URI anymore. that seems okay - could it maybe affect a plug-in that is defined with `uri = ""`? maybe...